src/video/cocoa/SDL_cocoakeyboard.m

```diff
@@ -448,7 +448,7 @@ void Cocoa_QuitKeyboard(_THIS)
 
 void Cocoa_SetWindowKeyboardGrab(_THIS, SDL_Window *window, SDL_bool grabbed)
 {
-#if SDL_MAC_NO_SANDBOX
+#if defined(SDL_MAC_NO_SANDBOX)
```

Comment:
Why if defined() and not ifdef?

Reply:
In general, we use ifdef for a single conditional, and if defined() for multiple conditionals or symmetry with another elif defined() block. It's not super important, it just stuck out and I wasn't sure if you had some guideline you were working with here.